src/core/exec-invoke.c

```diff
@@ -2548,6 +2557,96 @@ static int create_many_symlinks(const char *root, const char *source, char **sym
         return 0;
 }
 
+static int apply_exec_quotas(const char *target_dir, const char *cgroup_path, ExecDirectoryType type, QuotaLimit ql) {
+        assert(target_dir);
+        assert(ql.quota_set);
+
+        _cleanup_close_ int fd = -EBADF;
+        int r = 0;
+
+        fd = xopenat(AT_FDCWD, target_dir, O_RDONLY|O_CLOEXEC|O_NOCTTY|O_NOFOLLOW);
+        if (fd < 0)
+                return log_debug_errno(r, "Exec quotas: Failed to open '%s': %m", target_dir);
```

Comment:
xopenat() returns the error as negative ret val, and log_debug_errno() expects that as firts arg, hence change first arg to `fd` here